README.md

```diff
@@ -60,7 +60,76 @@ When creating a new entity definition you may use the following files as a guide
 
 For more concrete examples, you can take a look at the files located on the [definitions](./definitions) folder. 
 
-#### Conditions
+#### Synthesis
+
+The definition file may contain a synthesis section which is optional *only* if you can ensure that the telemetry is being stamped with the entity GUID and tags based on rules defined internally at NewRelic.
+
+If the telemetry you want to synthesize entities from always has the same attributes you can define a synthesis section with the following format:
```

Comment:
"You can define synthesis rules when your telemetry has consistent attributes that can uniquely identify an entity" seems clearer?

Reply:
👍  This is just for the simplified format, I'll reword it.